test/test_module_autogeneration.py

```diff
@@ -48,6 +48,7 @@
 from alignak_test import *
 
 
+@unittest.skipIf(True, "if you want namedpipe then define it in your config !")
```

Comment:
If you want to drop this feature say it explicitely. This is for Nagios copatibility actually, we can "debate" this point